### PR TITLE
Feature/public release

### PR DESCRIPTION
**Pull request by sschu**
Implements https://www.openproject.org/work_packages/3077

### CODE REVIEW

File: app/models/entry.rb [hunk 1]
```diff
@@ -1,5 +1,21 @@
-require_dependency "time_entry"
-require_dependency "cost_entry"
```
Comment (apfelfabrik) [inline app/models/entry.rb:2]: Is this removal intentional?

File: db/migrate/20130918084158_add_cost_object_journals.rb [hunk 2]
```diff
@@ -1,18 +1,11 @@
 #-- copyright
-# OpenProject is a project management system.
-# Copyright (C) 2012-2013 the OpenProject Foundation (OPF)
+# OpenProject Costs Plugin
 #
-# This program is free software; you can redistribute it and/or
-# modify it under the terms of the GNU General Public License version 3.
-#
-# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
-# Copyright (C) 2006-2013 Jean-Philippe Lang
```
Comment (apfelfabrik) [inline db/migrate/20130918084158_add_cost_object_journals.rb:9]: You are removing other copyright information here, is this intentional?

File: features/work_packages/destroy_with_cost_entries.feature [hunk 3]
```diff
@@ -1,18 +1,11 @@
...
-# modify it under the terms of the GNU General Public License version 3.
-#
-# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
-# Copyright (C) 2006-2013 Jean-Philippe Lang
-# Copyright (C) 2010-2013 the ChiliProject Team
+# Copyright (C) 2009 - 2014 the OpenProject Foundation (OPF)
 #
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
-# as published by the Free Software Foundation; either version 2
```
Comment (apfelfabrik) [inline features/work_packages/destroy_with_cost_entries.feature:14]: You are changing the license from GPL v2 to GPL v3. Is this intentional?

File: app/models/entry.rb (hunk 1 above)
Comment (sschu) [inline app/models/entry.rb:2]: Yes, both classes will be autoloaded 2 lines later.
